src/features/MentorPage/components/MentorsFilter/MobileSearch/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useTranslation } from 'react-i18next';
+
+import styled from 'styled-components';
+import { palette } from '@/components/variables';
+import { Text } from '@/components/Text/Text';
+import SearchBar from '@/components/SearchBar';
+
+type Props = {
+  searchString: string;
+  onSearchStringChange: (value: string) => void;
+};
+
+const MobileSearch = ({ searchString, onSearchStringChange }: Props) => {
+  const { t } = useTranslation('mentors');
+
+  return (
+    <MobileContainer>
+      <MobileHeader variant="h1">{t('filters.title')}</MobileHeader>
+      <Text>{t('filters.description')}</Text>
+      <SearchBar
+        variant="small"
+        placeholder="Etsi mentoria"
```

Comment:
Hmm this shouldn't be here. 😄 This was apparently missed when doing the localization work. Now it stays in Finnish even when the language is English.

Reply:
No, I added this and forgot to make a localization 8)

Reply:
Ah yes!